src/scripts/main.js

```diff
@@ -1,3 +1,330 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+const tHeadCols = Array.from(table.tHead.rows[0].cells);
+const tRows = Array.from(table.rows);
+const tFootCols = Array.from(table.tFoot.rows[0].cells);
+const tBodyRows = table.tBodies[0].rows;
+
+function salaryToNum(str) {
+  const arr = str.split(',');
+  const value = arr[0].slice(1) + arr[1];
+
+  return +value;
+}
+
+function salaryToDollars(value) {
+  let str = '';
+  let count = 1;
+  let res = '';
+
+  for (let i = value.length - 1; i >= 0; i--) {
+    if (count === 3 && i !== 0) {
+      str += value[i] + ',';
+      count = 1;
+    } else {
+      str += value[i];
+      count++;
+    }
+  }
+
+  for (let i = str.length - 1; i >= 0; i--) {
+    res += str[i];
+  }
+
+  return '$' + res;
+}
+
+function sortTable(e) {
+  const th = e.target.closest('th');
+  const indexThHead = Array.prototype.indexOf.call(tHeadCols, e.target);
+  const indexThFoot = Array.prototype.indexOf.call(tFootCols, e.target);
```

Comment:
The `indexThHead` and `indexThFoot` variables are calculated using `Array.prototype.indexOf.call`, which is correct but could be simplified using `tHeadCols.indexOf(e.target)` and `tFootCols.indexOf(e.target)` for better readability.